src/main/resources/js/admin_helper.js

```diff
@@ -62,7 +62,10 @@ function  checkPublicTokenAndOrganization(url) {
         var new_organization = AJS.$("#github_organization").val();
 
         var res = AJS.$.ajax({
-            url: "https://api.github.com/orgs/"+ new_organization +"/teams?access_token=" + new_private_token,
+            url: "https://api.github.com/orgs/"+ new_organization +"/teams",
+            headers: {
+                "Authorization": "token " + config.githubTokenPublic
```

Comment:
Token now evaluates to a different value. Are you sure that should not be `new_private_token`?

Reply:
changed call to private Token

Reply:
Please change to `new_private_token` so that we have the same behavior as before. Or am I missing something?